.golangci.yaml

```
@@ -135,6 +135,16 @@ linters-settings:
     - unnecessaryDefer
     - whyNoLint
     - wrapperFunc
+  tagliatelle:
+    case:
+      extended-rules:
+        json:
+          case: ''
```

We use camel case for the json tags so I think we should set it here
```suggestion
          case: 'camel'
```

---

I thought that too and tried it but for some odd reason then the initialism-overrides didn't work anymore (locally at least) and errors like `json(camel): got 'ipv6DHCP' want 'ipv6Dhcp' (tagliatelle)` returned.

---

Oh :open_mouth: 
Does the same happen if you have both camel case in both the rules and extended-rules?

---

Found out that that custom initialisms aren't supported: https://github.com/ldez/tagliatelle/issues/2. Too solve this I changed to use part of the initial solution + case: goCamel which includes some common initialisms